DataFormats/FTLDigi/interface/BTLSample.h

```diff
@@ -0,0 +1,95 @@
+#ifndef DIGIFTL_BTLSAMPLE_H
+#define DIGIFTL_BTLSAMPLE_H
+
+#include <iostream>
+#include <ostream>
+#include <cstdint>
+
+/**
+   @class BTLSample
+   @short wrapper for a data word
+ */
+
+class BTLSample {
+
+public:
+
+  enum BTLSampleDataMasks { kToA2Mask = 0x3ff, kToAMask = 0x3ff, kDataMask = 0x3ff};
```

Comment:
why are these enums instead of consts?

Reply:
I let @casarsa to confirm, but this looks just the minimal extension of the existing FTLSample, that was the target of the DIGI part of this PR. This could become a "static const uint32_t" as in many other DataFormats (including the BTLDetId and ETLDetId of this PR)

Reply:
I confirm. In the future we can rethink and improve this.